docs/pages/eas-update/migrate-codepush-to-eas-update.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Migrating from CodePush to EAS Update
+description: A guide to help migrate from CodePush to EAS Update.
+---
+
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+
+Transition a [bare](/bare/hello-world/) React Native project with CodePush to use EAS Update. This guide assumes that you are using the default React Native project structure.
+
```

Comment:
we dont have a pitch but ill get to it. will keep it separate from this pr to avoid bloat. tracking here https://linear.app/expo/issue/ENG-8954/eas-update-pitch